components/sections/NavItem/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { ArrowUpRightIcon } from '@heroicons/react/24/solid';
+import classNames from 'classnames';
+import type { FC } from 'react';
+import { useMemo } from 'react';
+
+import LocalizedLink from '@/components/LocalizedLink';
+
+import styles from './index.module.css';
+
+type NavItemType = 'nav' | 'footer';
+
+type NavItemProps = {
+  href: string;
+  label?: string;
+  type?: NavItemType;
+};
+
+const NavItem: FC<NavItemProps> = ({ href, label, type = 'nav' }) => {
+  const showIcon = useMemo(() => {
+    return href.startsWith('http') && type === 'nav';
```

Comment:
```tsx
const showIcon = useMemo(() => 
  type === 'nav' && /^https?:\/\//.test(href),
  [href, type]
);
```

Can we simplify this function to what I've shown above? This regex is also what we use on the `LocalizedLink` component.

Reply:
updated